tests/orquestra/quantum/operators/_operators_test.py

```diff
@@ -641,3 +641,14 @@ def test_paulisum_repr(self, pauli_sum):
     def test_paulisum_multiply_by_zero(self, pauli_sum: PauliSum):
         zero_op = pauli_sum * 0
         assert zero_op == PauliSum()
+
+    def test_paulisum_constant_value_zero(self, pauli_sum):
+        assert 0 == pauli_sum.constant_value
+
+    def test_paulisum_constant(self, pauli_sum):
```

Comment:
Please do parameterize with other values as well – namely non-integers and negative values.

Reply:
Done